### PR TITLE
Feature/mast 3 create login register page

### PR DESCRIPTION
**Pull request by MichalKyjovsky**

### CODE REVIEW

**Review by TomasDrozdik — DISMISSED** [review]
Seems fine.

File: mast/templates/login.html [hunk 1]
```diff
@@ -1,10 +1,44 @@
...
+                <div class="form-label-group">
+                    <input type="password" id="inputPassword" class="form-control" placeholder="Password" required>
+                    <label for="inputPassword">Password</label>
+                </div>
+
+                <div class="checkbox mb-3">
+                    <label>
+                        <input type="checkbox" value="remember-me"> Remember me
+                    </label><br>
+                    <a href="{{ url_for( 'register' ) }}">Sign-up</a>
```
Comment (peterkorcsok) [inline mast/templates/login.html]: I'd rather avoid using "Sign in" and "Sign up" - could be confusing a little bit :)

Suggestion: "Create a new account" / "Are you new?"

File: mast/templates/register.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    </div>
+
+    <div class="form-label-group">
+        <input type="password" id="confirmPassword" class="form-control" placeholder="Confirm Password" required>
+        <label for="confirmPassword">Confirm Password</label> 
+    </div>
+
+    <div class="checkbox mb-3">
+        <!-- TODO: Add support for 3-rd party applications. -->
+        <a href="{{ url_for( 'login' ) }}">Already Signed-in?</a>
```
Comment (peterkorcsok) [inline mast/templates/register.html]: "Signed in" = I successfully entered credentials and I was allowed to enter :D

Suggestion: "Already registered?" / "Do you already have an account? Go to Sign-in page"

File: mast/templates/register.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    </div>
+    <button class="btn btn-lg btn-primary btn-block" type="submit">Sign up</button>
```
Comment (peterkorcsok) [inline mast/templates/register.html]: Same as in login.html

Suggestion: "Register" / "Create an account"

**Review by peterkorcsok — CHANGES_REQUESTED** [review]

**Review by peterkorcsok — APPROVED** [review]
Job well done

**Pull request merged**